testutil/compose/compose/smoke_internal_test.go

```diff
@@ -86,24 +86,8 @@ func TestSmoke(t *testing.T) {
 				conf.KeyGen = compose.KeyGenCreate
 			},
 		},
-		// TODO(dhruv): enable version matrix tests for dkg once we have compatible versions
-		//{
-		//	Name:     "run version matrix with dkg",
-		//	PrintYML: true,
-		//	ConfigFunc: func(conf *compose.Config) {
-		//		conf.KeyGen = compose.KeyGenDKG
-		//	},
```

Comment:
I think uncomment is better than removing since we want to re-enable very soon

Reply:
sure